### ISSUE TITLE
Policy extractor code loops for every policy parameter

### ISSUE THREAD

**Issue by phochste** [eyereasoner/Koreografeye#7]
https://github.com/eyereasoner/Koreografeye/blob/main/src/policy/Extractor.ts#L46

Take policy:

```
    ex:MySendNotificationDemo pol:policy [
        a fno:Execution ;
        fno:executes ex:sendNotification ;
        ex:to <http://patrickhochstenbach.net/inbox/> ;
        ex:notification [
                a as:Accept ;
                as:inReplyTo ?id ;
                as:actor     <http://patrickhochstenbach.net/profile/card#me> ;
                as:origin    <http://patrickhochstenbach.net/orchestrator/profile/card#me> ;
                as:target    ?actor;
                as:object    ?id
        ]
    ] .
```

The `ex:sendNotification` as two parameters `ex:to` and `ex:notification` this means that block L46 is executed two times. Refactor this into one loop.

**Comment by phochste** [comment]
There is too much and too little looping in the code. When there are multiple ?id-s available only one execution of ex:MySendNotification will be run. This is for some time documented as a bug.

**Comment by phochste** [comment]
Fixed in version 0.4.0
